functions/api/branch.js

```diff
@@ -189,7 +189,7 @@ router.delete('/:id', isSuperAdmin, async (request, response, next) => {
     await branchRef.delete()
 
     // Add event in Audit
-    const eventMessage = `User ${request.user.firstName} deleted branch ${branchData[constants.NAME]}`
+    const eventMessage = `User ${req.user.name} deleted branch ${branchData[constants.NAME]}`
```

Comment:
same doubt, is it req or request ?

Reply:
good catch thanks. I tried to do find, replace - it kindof screwed it up